README.md

```diff
@@ -73,6 +73,8 @@ export class AppHomeComponent {
 
 - `autoUpload` Auto start upload when files added. Default value: `true`
 
+- `storeIncompleteUploadUrl` Keep an incomplete upload URL for allow resuming after browser restart. Default value: `true`
```

Comment:
Personally, I think that a more appropriate name would be `autoResume` (consistently with `autoUpload`. The name `storeIncompleteUploadUrl` is too much depending on the implementation detail, rather than what the end-result is.

Reply:
`autoResume` is perhaps too promising as `autoUpload` 😊
 I don't like ёstoreIncompleteUploadUrlё either